src/components/donation/Donation.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { formatNumber } from '../shared/utils';
+
+interface Props {
+  total: number;
+  goal: number;
+}
+
+export const Donation: React.StatelessComponent<Props> = (props: Props) => {
+  const pctDone = (props.total / props.goal) * 100;
+  const pctDoneStyle = {width: `${pctDone}%`};
+  return (
```

Comment:
One thing to change here: if goal isn't set, don't show the donation component at all. See the difference between this implementation (flashes $0/$1) versus currently on 5calls.org (doesn't exist, then shows up)